app/MMB/src/scenes/appleWallet/Passenger.js

```diff
@@ -0,0 +1,25 @@
+// @flow strict
+
+import * as React from 'react';
+import { Translation } from '@kiwicom/mobile-localization';
+
+import WalletText from './WalletText';
+import WalletContext from '../../context/WalletContext';
+
+export default function Passenger() {
+  return (
+    <WalletContext.Consumer>
+      {({ selectedSegment }) => {
+        const walletData = selectedSegment || {};
+        return (
+          <WalletText
+            label={
+              <Translation id="mmb.apple_wallet.apple_wallet_scene.passenger" />
+            }
+            text={<Translation passThrough={walletData.passengerName} />}
```

Comment:
This will render undefined if selectedSegment is falsy, right?

Reply:
yes, that is correct, I should fix that